src/models/machineInitializerToken.ts

```diff
@@ -0,0 +1,28 @@
+
+export interface MachineInitializerTokenData {
+  initializerToken: string
+  url: string
+  expiresAt?: string
+}
```

Comment:
Make 2 different interfaces with explicit declaration for `expiresAt`. (One can inherit from the other).